add-ons/istio/README.adoc

```diff
@@ -4,12 +4,12 @@
 
 This add-on deploys the https://Istio.io/[Istio] **servicemesh** on to Minishift-provisioned OpenShift cluster.
 
-NOTE: The add-on was tested with Istio release v0.3.0
+NOTE: The add-on was tested with Istio release v1.0.1
```

Comment:
@literalice looks like you forgot to update the version here, I think you are doing it for 1.0.0 instead of 1.0.1 because of selinux stuff.

Reply:
Oh.. Thank you for check it. I've fixed it.